[2022-08-31 14:34:16,759] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: green_taxi_data_ingest.local_to_gcs_task manual__2022-08-31T14:34:02+00:00 [queued]>
[2022-08-31 14:34:16,767] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: green_taxi_data_ingest.local_to_gcs_task manual__2022-08-31T14:34:02+00:00 [queued]>
[2022-08-31 14:34:16,767] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-08-31 14:34:16,767] {taskinstance.py:1239} INFO - Starting attempt 1 of 1
[2022-08-31 14:34:16,767] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-08-31 14:34:16,780] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): local_to_gcs_task> on 2022-08-31 14:34:02+00:00
[2022-08-31 14:34:16,785] {standard_task_runner.py:52} INFO - Started process 4815 to run task
[2022-08-31 14:34:16,787] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'green_taxi_data_ingest', 'local_to_gcs_task', 'manual__2022-08-31T14:34:02+00:00', '--job-id', '172', '--raw', '--subdir', 'DAGS_FOLDER/green_taxi_data_ingest_dag.py', '--cfg-path', '/tmp/tmp0pxmh_l0', '--error-file', '/tmp/tmpmg2jxxb2']
[2022-08-31 14:34:16,787] {standard_task_runner.py:77} INFO - Job 172: Subtask local_to_gcs_task
[2022-08-31 14:34:16,835] {logging_mixin.py:109} INFO - Running <TaskInstance: green_taxi_data_ingest.local_to_gcs_task manual__2022-08-31T14:34:02+00:00 [running]> on host 0aad59a5352b
[2022-08-31 14:34:16,886] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=green_taxi_data_ingest
AIRFLOW_CTX_TASK_ID=local_to_gcs_task
AIRFLOW_CTX_EXECUTION_DATE=2022-08-31T14:34:02+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-08-31T14:34:02+00:00
[2022-08-31 14:34:18,284] {python.py:175} INFO - Done. Returned value was: None
[2022-08-31 14:34:18,294] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=green_taxi_data_ingest, task_id=local_to_gcs_task, execution_date=20220831T143402, start_date=20220831T143416, end_date=20220831T143418
[2022-08-31 14:34:18,324] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-08-31 14:34:18,354] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
